lib/property-exists.js

```diff
@@ -0,0 +1,13 @@
+module.exports = function checkNestedProperty (obj, pathString) {
+  var path = pathString.split('.');
+
+  var property = path.reduce((prev, curr) => {
+    if (prev[curr]) {
+      return prev[curr];
+    } else {
+      return false;
+    }
+  }, obj);
+
+  return property;
+};
```

Comment:
I'd be tempted to rename this now since it's no longer doing an existence check, but one for another PR perhaps :)